python/tvm/autotvm/task/relay_integration.py

```diff
@@ -103,8 +104,8 @@ def extract_from_multiple_program(funcs, params, ops, target, target_host=None,
         The compilation target
     target_host: tvm.target.Target
         The host compilation target
-    template_key: str
-        The tuning template key for schedules, default to `direct`
+    template_keys: direct of topi op to str
+        The tuning template keys for schedules, default to None.
```

Comment:
The default value of this argument is not `None` actually. On the other hand, I would suggest using `None` as the default, because an empty container is not recommended [[ref]](https://docs.python.org/3/reference/compound_stmts.html#function-definitions)

Reply:
Not sure why I saw `template_keys=dict()`. Anyway, the current one is fine.